packages/app-desktop/gui/MainScreen/commands/gotoAnything.ts

```diff
@@ -27,7 +27,35 @@ export const runtime = (): CommandRuntime => {
 			if (uiType === UiType.GotoAnything) {
 				menuItemById('gotoAnything').click();
 			} else if (uiType === UiType.CommandPalette) {
-				menuItemById('commandPalette').click();
+				const originalMenuItem = menuItemById('commandPalette');
```

Comment:
Every time the command is called, it adds a new handler, am I reading this correctly? Looks like it has to be done only once (and probably not here)